### PR TITLE
feat(website/test): create separated html for each preview (and display it in previews iframe)

### PR DESCRIPTION
**Pull request by mohamedMok**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [X] Yes
- [ ] No

## Does this PR introduce a breaking change?

read [What is a breaking change ?](https://mozaic.adeo.cloud/Contributing/Prerequisite/GitConventions/#breaking-changes-)

- [ ] Yes
- [X] No

## Describe the changes

Github issue number or Jira issue URL:
https://design-system-adeo.atlassian.net/browse/MZC-194

### CODE REVIEW

File: packages/gatsby-theme-styleguide/src/templates/preview-page.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { graphql } from 'gatsby'
+
+export default ({ data }) => {
+  return (
+    <>
```
Comment (gael-boyenval) [inline packages/gatsby-theme-styleguide/src/templates/preview-page.js]: We should use `Helmet` to add styles in the `head`, and is possible inject the html inside the `body` tag

File: packages/gatsby-theme-styleguide/src/gatsby-components/Preview/PreviewFrame.js [hunk 2]
```diff
@@ -111,48 +112,12 @@ export class PreviewFrame extends PureComponent {
...
-    })
-  }
-
   render() {
     const { iframeHeight } = this.state
     const { viewport, viewPorts, availableWidth, fullScreen, grid } = this.props
-
+    const iframeSrc = this.props.data.node.path
+      ? `${window.location.hostname}/${this.props.data.node.path}`
+      : ''
```
Comment (gael-boyenval) [inline packages/gatsby-theme-styleguide/src/gatsby-components/Preview/PreviewFrame.js]: maybe just : 

```js 
this.props.data.node.path && window
     ? `${window.location.hostname}/${this.props.data.node.path}`
     : ''
```

Comment (mohamedMok) [reply packages/gatsby-theme-styleguide/src/gatsby-components/Preview/PreviewFrame.js]: Load window in componentDidMount function

File: packages/gatsby-theme-styleguide/src/templates/preview-page.js (hunk 1 above)
Comment (mohamedMok) [reply packages/gatsby-theme-styleguide/src/templates/preview-page.js]: Helmet added

**Pull request merged**